packages/web-components/src/components/footer/footer.scss

```diff
@@ -397,7 +397,7 @@
     margin-top: -#{$carbon--spacing-05};
   }
 
-  @include carbon--breakpoint('lg') {
+  @include carbon--breakpoint-down('lg') {
```

Comment:
Hum, this isn't quite it. I'm now seeing no language selector display for anything under the `lg` breakpoint.

The other issue is that `carbon--breakpoint-down` produces a media query like `(max-width: 65.98rem)`, which leaves out the border pixel. I think either you change the CSS to never include `display: none` b/c the precence of either tag is now controlled in Javascript. Or you have `:host(#{$dds-prefix}-language-selector-desktop)` default to `display: none;` with a media query using `carbon--breakpoint('lg')` to set it to `display: block`. Then do the opposite for `:host(#{$dds-prefix}-language-selector-mobile)`.

Reply:
Oof, sorry @m4olivei. That was sloppy on my end. I removed the CSS responsible for hiding these elements, instead relying on the JS to do that work. I promise I double-checked this time, too 😅

Reply:
All good! It's looking good to me now. Thanks.